src/Features/LanguageServer/Protocol/Features/Diagnostics/DiagnosticService.cs

```diff
@@ -197,169 +195,6 @@ private void OnCleared(object? sender, EventArgs e)
             RaiseDiagnosticsCleared((IDiagnosticUpdateSource)sender!);
         }
 
-        public ValueTask<ImmutableArray<DiagnosticData>> GetDiagnosticsAsync(
```

Comment:
no actual product code called the two public methods below.  removed them and all tehir supporting helpers.